packages/twenty-front/src/pages/onboarding/CreateProfile.tsx

```diff
@@ -43,7 +43,7 @@ const StyledComboInputContainer = styled.div`
   display: flex;
   flex-direction: row;
   > * + * {
-    margin-left: ${({ theme }) => theme.spacing(4)};
+    margin-inline-start: ${({ theme }) => theme.spacing(4)};
```

Comment:
interesting, this might be a big change in the code base. would be great to discuss impacts with @Bonapara

Reply:
@charlesBochet not sure what’s implied here